src/s1a_portapotty.tw2

```diff
@@ -88,61 +66,66 @@ I reached forward, snatched it up, and recognized its familiar heft immediately.
 
 I gave it a good flick, and the room sprang into detail around me.
 
-[[* click *|Jail Cell (Lit) - hub]] ]\
-\
-(click: ?trigger1)[(show: ?section2)] \
-(click: ?trigger2)[(show: ?section3)] \
-(click: ?trigger3)[(show: ?section4)] \
-(click: ?trigger4)[(show: ?section5)] \
-(click: ?trigger5a)[(replace: ?prompt5b)[] (show: ?section6)] \
-(click: ?trigger5b)[(replace: ?prompt5a)[] (show: ?section6)]
-
-::Jail Cell (Unlit) - look - fail <30,500>
-My eyelids felt like paperweights. My eyes were as dry as sandpaper. I couldn't see shit.
-
-[[I tried rubbing my face. No good. Fuck.|Jail Cell (Unlit) - hub]]
-
-::Jail Cell (Unlit) - hub <202,344>
-Everything around me was darkness. (if: $sniffedAir is true)[
-
-Well...darkness and shit.]
-
->> [[I blinked my eyes, still heavy from sleep.|Jail Cell (Unlit) - look - fail]]
->> [[I felt around for something concrete.|Jail Cell (Unlit) - feel]]
->> [[I sniffed the stagnant air.|Jail Cell (Unlit) - smell]]
+[[`*` click `*`|Jail Cell (Lit) - hub]] ]\
+{
+(click: ?trigger1)[(show: ?section2)]
+(click: ?trigger2)[(show: ?section3)]
+(click: ?trigger3)[(show: ?section4)]
+(click: ?trigger4)[(show: ?section5)]
+(click: ?trigger5a)[(replace: ?prompt5b)[] (replace: ?prompt5a)[a faint beam of moonlight draped across the cell walls.] (show: ?section6)]
+(click: ?trigger5b)[(replace: ?prompt5a)[] (replace: ?prompt5b)[a glint of metal on the floor.] (show: ?section6)]
+(set: $triedDoor to false)
+(set: $waterDrained to false)
+(set: $hasJailKey to false)
+}
 
 ::Jail Cell (Unlit) - smell <362,502>
-Shit. That's all I could pick up on. Someone literally lost their shit in here.
+Shit. That’s all I could pick up on. Someone literally lost their shit in here.
 
-I panicked—I couldn't tell where it was coming from. Like some Edgar Allan Poe thing—a Telltale Shit, lurking under the floor.
+I panicked, because I still couldn’t tell where it was. And now, like some Edgar Allan Poe thing — there was a Telltale Shit lurking under the floor.
 
->> [[I held my breath and tried not to vom all over myself.|Jail Cell (Unlit) - hub]]
+[[I held my breath and tried not to vom all over myself.|Jail Cell (Unlit) - hub]]
 
 (set: $sniffedAir to true)
 
 ::Jail Cell (Lit) - hub <201,659>
-The lighter's dancing flame illuminated my surroundings. (if: $triedDoor is 1)[I didn't like what I saw.]
+The lighter's dancing flame illuminated my surroundings. (if: $triedDoor is true)[I didn't like what I saw.]
```

Comment:
"I saw" ends the sentence here and starts the next one.

Reply:
Good catch. I'll replace the following instance with "There was".